### DISCUSSION TITLE
Add "who is using daisyui" to README

### DISCUSSION THREAD

**Discussion by arkxfly** [saadeghi/daisyui#2289] [Ideas / feature requests]
As a newbee，see such beautiful components of daisyui. But I don't know how to better apply it to my own website.

Hope there is "who is using daisyui" on README.

**Comment by saadeghi** [comment]
I will add a section to the website for that.  
However I think you can also get inspiration from any UI design. For example you can find a good UI design on Dribbble or other design sources and use daisyUI components to recreate it.

**Reply by arkxfly** [reply]
Thanks!!! Dribbble is very good.
